packages/devtools_app/lib/src/screens/vm_developer/object_viewport.dart

```diff
@@ -84,9 +81,8 @@ String _getViewportTitle(ObjRef objRef) {
 //Calls the object VM statistics card builder according to the VM Object type.
 Widget _buildObjectScreen(ObjRef objRef) {
   if (objRef is ClassRef) {
-    final classController = ClassScreenController(objRef);
     return VmClassScreen(
-      controller: classController,
+      controller: ClassScreenController(objRef),
```

Comment:
I'm wondering if it'd just be better to pass `objRef` directly into `VmClassScreen` and let it handle creating the controller internally since there's currently no need for the controller to be accessible outside of the screen.